components/feature/addons/src/main/java/mozilla/components/feature/addons/ui/CustomViewHolder.kt

```diff
@@ -0,0 +1,45 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.feature.addons.ui
+
+import android.view.View
+import android.widget.ImageView
+import android.widget.RatingBar
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+
+/**
+ * A base view holder.
+ */
+sealed class CustomViewHolder(view: View) : RecyclerView.ViewHolder(view) {
```

Comment:
nit: If we don need this class outside of `feature.addons.ui` we could make it internal.

Reply:
This can't be internal since the linter will complain that AddonsAdapter is exposing an internal supertype in RecyclerView.Adapter<CustomViewHolder>().